### ISSUE TITLE
Update automation logic when metagenome workflow process starts from annotation

### ISSUE THREAD

**Issue by aclum** [microbiomedata/nmdc_automation#35]
IMG is having issues ingesting our re-annotation of JGI assemblies b/c the contigs are named based on JGI assembly group (scaffold_1_c1) and the annotation files use nmdc naming convention without a conversion file (nmdc:wfmgan-11-2r9bkx60.1_1_c1)
example omics processing directory nmdc:omprc-11-14ermv40

I believe this can be fixed by specifying pre_qc_execute=true to the structural-annotation.wdl.  I'm making a related ticket in the mg_annotation repo to make sure this works correctly. Automation code will need to make a mongo record for the mapping file (*_contig_names_mapping.tsv). This file already has an enumeration in the schema 'Contig Mapping File'. That mapping file then needs to be passed to the MAGs workflow.

acceptance criteria:

From a workflow run starting with a JGI assembly as input, there is a directory on the file system which contains a mapping file with extension `_contig_names_mapping.tsv` with a corresponding DataObject mongo with data_object_type of `Contig Mapping File`. That mapping file is correctly passed to [MagsAnalysisActivity](https://microbiomedata.github.io/nmdc-schema/MagsAnalysisActivity/) and that activity completes successfully.

**Comment by aclum** [comment]
@Michal-Babins  made a good point that this should only apply if the assembly was not generated by NMDC. You could look at processing_institution for the assembly component for the logic statement.

**Comment by aclum** [comment]
backlogging this in favor of issues https://github.com/microbiomedata/issues/issues/551 and re-iding work.

**Comment by aclum** [comment]
@chienchi  can you make a new release of metaMAGs with the fixes to provide the mapping file so the automation logic can test updating to using that version to optionally supply a mapping file.

**Comment by aclum** [comment]
Chienchi made a new release for metagenomes, we are still blocked on https://github.com/microbiomedata/mg_annotation/issues/24

**Comment by aclum** [comment]
Duplicate of https://github.com/microbiomedata/nmdc_automation/issues/191
